src/screens/Resources/ResourcesItemListScreen.js

```diff
@@ -0,0 +1,183 @@
+import React, { useState } from 'react';
+import {
+  ScrollView, View, TouchableOpacity, StyleSheet,
+} from 'react-native';
+import PropTypes from 'prop-types';
+import {
+  Text, ActivityIndicator,
+} from 'react-native-paper';
+import Collapsible from 'react-native-collapsible';
+import Accordion from 'react-native-collapsible/Accordion';
+import Icon from 'react-native-vector-icons/FontAwesome';
+import { NavigationEvents } from 'react-navigation';
+import { theme } from '../../style';
+import { nav } from './variables';
+
+export default function ResourcesItemListScreen(props) {
+  const { navigation } = props;
+  const snapshot = navigation.getParam('snapshot');
+  const [loading, setLoading] = useState(false);
+  const [activeSections, setActiveSections] = useState([]);
+  const [singleActive, setSingleActive] = useState(false);
+
+  function renderHeader(doc, _, isActive) {
+    return (
+      <View style={ItemListStyles.accordionHeader}>
+        <View style={ItemListStyles.accordionHeaderText}>
+          <Text style={ItemListStyles.resourceText}>
+            {doc.data().title}
+          </Text>
+        </View>
+        <View style={ItemListStyles.accordionHeaderIcon}>
+          <Icon name={isActive ? 'chevron-up' : 'chevron-down'} />
+        </View>
+      </View>
+    );
+  }
+
+  function renderContent(doc) {
+    return (
+      <View style={ItemListStyles.accordionContent}>
+        <Text>
+          {doc.data().description}
+        </Text>
+
+        <View style={ItemListStyles.moreButton}>
+          <Text
+            style={ItemListStyles.resourceText}
+            onPress={() => {
+              setLoading(true);
+              navigation.push(nav.item, { resourceID: doc.id });
+            }}
+          >
+            More
+            {' '}
+            <Icon name="chevron-right" />
+          </Text>
+        </View>
+      </View>
+    );
+  }
+
+  function updateSections(currActive) {
+    setActiveSections(currActive);
+  }
+
+  function singleOrMultipleDocs() {
+    // returns a collapsible if there is only 1 doc
+    if (snapshot.docs.length === 1) {
```

Comment:
What if snapshot is empty?

Reply:
if snapshot is empty it works fine, there's just no interactive parts to the screen.